src/main/resources/anniv5Resources/localization/eng/metapack/Powerstrings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "${ModID}:RayOfFrostPower": {
+    "NAME": "Magical Protection",
+    "DESCRIPTIONS": [
+      "Channel #b",
+      " Frost whenever you play a Magic card this turn."
```

Comment:
```suggestion
      " #yFrost whenever you play a #yMagic card this turn."
```

Reply:
Are you sure? They are already highlighted.
![image](https://user-images.githubusercontent.com/92187869/211158618-0ea09273-8697-43e5-bdc2-e013b8edb741.png)

Reply:
This is for the power